packages/ui/src/components/Header/index.tsx

```diff
@@ -12,22 +12,18 @@ import { isEmptyArray } from '../../utils'
 import NetworkSelection from '../NetworkSelection'
 
 interface Props {
-  className?: string
   handleDrawerOpen: () => void
 }
 
-const Header = ({ className, handleDrawerOpen }: Props) => {
+const Header = ({ handleDrawerOpen }: Props) => {
   const { ownAccountList } = useAccounts()
   const { multiProxyList } = useMultiProxy()
   const isAccountConnected = useMemo(() => !isEmptyArray(ownAccountList), [ownAccountList])
   const isAtLeastOneMultiProxy = useMemo(() => !isEmptyArray(multiProxyList), [multiProxyList])
   const { isAllowedToConnectToExtension, allowConnectionToExtension } = useAccounts()
 
   return (
-    <MuiAppBar
-      position="fixed"
-      className={className}
-    >
+    <MuiAppBar position="sticky">
```

Comment:
this changed the UI quite a bit, it's off now
![image](https://github.com/ChainSafe/Multix/assets/33178835/97437a91-ff63-4502-a081-181ed2134507)